pkg/scheduler/scheduler.go

```diff
@@ -250,22 +229,14 @@ var defaultSchedulerOptions = schedulerOptions{
 	schedulerAlgorithmSource: schedulerapi.SchedulerAlgorithmSource{
 		Provider: defaultAlgorithmSourceProviderName(),
 	},
-	hardPodAffinitySymmetricWeight:  v1.DefaultHardPodAffinitySymmetricWeight,
-	disablePreemption:               false,
-	percentageOfNodesToScore:        schedulerapi.DefaultPercentageOfNodesToScore,
-	bindTimeoutSeconds:              BindTimeoutSeconds,
-	podInitialBackoffSeconds:        int64(internalqueue.DefaultPodInitialBackoffDuration.Seconds()),
-	podMaxBackoffSeconds:            int64(internalqueue.DefaultPodMaxBackoffDuration.Seconds()),
-	frameworkConfigProducerRegistry: frameworkplugins.NewConfigProducerRegistry(),
-	// The plugins and pluginConfig options are currently nil because we currently don't have
```

Comment:
Is the comment still applicable?

Reply:
nope, removed.

Reply:
The previous comment says the parameters should be populated once we have native plugins. Can you populate the parameters now?

Reply:
This wasn't the right place to populate them, the new comment clarifies that those parameters only carry the plugins specified in Component Config.

Reply:
The default plugins are now populated when calling "createFromProvider".

Reply:
Ack. Can we just remove these two lines (using go default values) then?

Reply:
done.